### ISSUE TITLE
silverstripe/silverstripe-framework 4.0.0-beta1

### ISSUE THREAD

**Issue by WillGibson** [markguinn/silverstripe-email-helpers#25]
I cannot install this at the same time as silverstripe/silverstripe-framework 4.0.0-beta1 because this wants silverstripe/silverstripe-framework ~3.0.

Can this be resolved?

**Comment by sminnee** [comment]
We'll need to make a SilverStripe 4 compatible version of this module to resolve that, it would be a new major release.

**Comment by sminnee** [comment]
Note that SS4 now makes use of Swift Mailer, so it's not clear how much that necessitates refactoring this module, or whether it's been replaced by Swift Mailer features.

**Comment by bummzack** [comment]
A lot of the functionality of this module will be replaced by SwiftMailer. The only thing I can think of that needs to be ported would be the emogrifier part… we use this in shop to inline the CSS styles. Not sure if this warrants it's own module though, it would certainly need some restructuring and integration with SwiftMailer.

@markguinn is no longer maintaining these modules (correct me if I'm wrong, Mark), but I guess he would still merge PRs.

**Comment by markguinn** [comment]
Yeah, unfortunately I changed jobs and haven't been working in PHP for a while. I've been a little negligent on merging PR's because it's difficult for me to test things locally and the farther I get from working in this stuff the harder it is to just look at the code and insure it's right. 

That said, I'll do my best to keep up if y'all submit them and I'm happy to hand the module off if anyone wants to pick it up.

**Comment by sminnee** [comment]
I've created some "additional committers?" tickets on the 3 modules that are silvershop dependencies. :-)

**Comment by xini** [comment]
The emogrifier part of this module can be replaced with `bummzack/silverstripe-emogrify ^0.2`.
The rest should be taken care of by SwiftMailer.
I think this issue can be closed.
